web/src/components/Breadcrumb.tsx

```diff
@@ -1,48 +1,40 @@
+import Button from '@mui/material/Button'
+import Typography from '@mui/material/Typography'
 import { styled } from '@mui/material/styles'
-import React, { ReactElement, ReactNode } from 'react'
+import React, { ReactElement } from 'react'
+import { useLocation } from 'react-router-dom'
 
-import { helpers } from '../constants/theme'
+import { normalizePath } from 'shared'
 
-const SHRINK_FACTOR = 0.1
-const ListItem = styled('li')<{ shrink: boolean }>`
-  overflow: hidden;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  flex-shrink: ${props => (props.shrink ? SHRINK_FACTOR : 0)};
+import Link from './base/Link'
 
-  &:not(:last-of-type) {
-    flex-shrink: 1;
-  }
-
-  & * {
-    ${helpers.removeLinkHighlighting}
-    color: ${props => props.theme.legacy.colors.textColor};
-    font-size: 16px;
-    margin: 0 2px;
-  }
-`
+const StyledButton = styled(Button)({
+  width: '100%',
+}) as typeof Button
 
-const Separator = styled('span')`
-  &::before {
-    color: ${props => props.theme.legacy.colors.textColor};
-    font-size: 19px;
-    content: ' > ';
-  }
-`
+const StyledTypography = styled(Typography)({
+  overflow: 'hidden',
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+})
 
 type BreadcrumbProps = {
-  children: ReactNode
-  shrink: boolean
+  title: string
+  to: string
 }
 
-/**
- * Displays breadcrumbs (Links) for lower category levels
- */
-const Breadcrumb = ({ children, shrink }: BreadcrumbProps): ReactElement => (
-  <ListItem shrink={shrink}>
-    <Separator aria-hidden />
-    {children}
-  </ListItem>
-)
+const Breadcrumb = ({ title, to }: BreadcrumbProps): ReactElement => {
+  const current = to === normalizePath(useLocation().pathname)
+
+  if (current) {
+    return <StyledTypography aria-current='page'>{title} </StyledTypography>
```

Comment:
```suggestion
    return <Link to={to} aria-current='page'>{title}</Link>
```

Reply:
1- Even if we show just 1 breadcrumb for small screens the title could be too long:
http://localhost:9000/muenchen/de/beratung-und-hilfe-4/beratung-und-hilfe-allgemein/formularservice-hilfe-beim-ausf%C3%BCllen-von-formularen-und-antr%C3%A4gen

<img width="400" alt="image" src="https://github.com/user-attachments/assets/014c7fbb-32a9-442b-bc4e-ae75053399f7" />


2- Did we agreed to keep the last breadcrumb non-Clickable ?